client/app/validations/saveable-pas-form.js

```diff
@@ -0,0 +1,104 @@
+import {
+  validateLength,
+} from 'ember-changeset-validations/validators';
+
+// These validate the fields for _saving_ to the server
+export default {
+  dcpRevisedprojectname: [
+    validateLength({
+      max: 50,
+      message: 'Name must be between {min} and {max} characters',
+    }),
+  ],
+
+  dcpDescriptionofprojectareageography: [
+    validateLength({
+      max: 2000,
+      message: 'Must be between {min} and {max} characters',
```

Comment:
Since a min isn't specified, does it default to plugging in zero instead?

Reply:
Hmm... I think this is referring to character length? It'll just be a blank string.